--- ec2.tf.bak	2020-12-13 15:27:47.000000000 +0900
+++ ec2.tf	2020-12-13 15:28:01.000000000 +0900
@@ -1,7 +1,7 @@
 resource "aws_instance" "web_servers" {
   count         = length(var.availability_zones)
   ami           = "ami-0053d11f74e9e7f52"
-  instance_type = "m5.large"
+  instance_type = "t3.micro"
   subnet_id     = aws_subnet.private_subnet[count.index].id
   tags = {
     Name = format("%s-%s-web-server-%s", var.application_name, var.environment, var.availability_zones[count.index])
